datajoint/expression.py

```diff
@@ -106,9 +106,7 @@ def make_sql(self, fields=None):
         Make the SQL SELECT statement.
         :param fields: used to explicitly set the select attributes
         """
-        distinct = self.heading.names == self.primary_key
-        return 'SELECT {distinct}{fields} FROM {from_}{where}'.format(
-            distinct="DISTINCT " if distinct else "",
+        return 'SELECT {fields} FROM {from_}{where}'.format(
```

Comment:
Can you also explore all usage of `DISTINCT`. Noticed that there is another similar usage in the `Aggregation` class. Did you mean to remove its use here but maintain its use there?

Reply:
I looked through it a bit and saw that it also uses `DISTINCT`, but it might be best to just leave it until a user reports a specific issue that is related to the aggregation usage of distinct. It would be a bad idea to remove it without a specific reason/issue.